plugins/filter/time.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, René Moser <mail@renemoser.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleFilterError
+
+
+UNIT_FACTORS = {
```

Comment:
How about having two tables (one for parameters to `to_time_unit`, one for time units in human-readable strings), and always representing everything in milliseconds? (I'd also add support for milliseconds BTW.)

Then the tables are shorter and it is easier to add a new unit (+2 entries instead of +1+n entries).

Reply:
makes sense.